quarkus/config-api/src/main/java/org/keycloak/config/HttpOptions.java

```diff
@@ -63,6 +63,17 @@ public enum ClientAuth {
             .defaultValue(Arrays.asList("TLSv1.3,TLSv1.2"))
             .build();
 
+    public static final Option<String> HTTPS_CERTIFICATES_RELOAD_PERIOD = new OptionBuilder<>("https-certificates-reload-period", String.class)
+            .category(OptionCategory.HTTP)
+            .description("Interval on which to reload key store, trust store, and certificate files referenced by https-* options. May be a java.time.Duration value, an integer number of seconds, or an integer followed by one of [ms, h, m, s, d]. Must be greater than 30 seconds. Use -1 to disable.")
+            .defaultValue("1h")
+            .build();
+    
+    public static final Option<String> HTTPS_CERTIFICATES_RELOAD_PERIOD_INTERNAL = new OptionBuilder<>("https-certificates-reload-period-internal", String.class)
```

Comment:
@shawkins I think it's not necessary to have it here, right?

Reply:
It is not - I was temporarly going to try another approach that didn't touch the property mappers, but abandoned that.